lua/smh/modifiers/bodygroup.lua

```diff
@@ -21,5 +21,7 @@ function MOD:Load(player, entity, data)
 end
 
 function MOD:LoadBetween(player, entity, data1, data2, percentage)
-    self:Load(player, entity, data1);
+	if not player.SMHData.TweenDisable then
```

Comment:
Instead of adding this condition to all of the modifiers, you can put it in [positioning.lua](https://github.com/Winded/StopMotionHelper/blob/master/lua/smh/server/positioning.lua#L79). This way, it will be applied to all modifiers, reducing duplicate logic by a lot.